src/backend/shared/styles/globals.css

```
@@ -258,6 +258,506 @@
+  box-shadow:
+    inset -1px -1px #fff,
+    inset 1px 1px #0a0a0a,
+    inset -2px -2px #dfdfdf,
+    inset 2px 2px #808080 !important;
+}
+
+.nineties button:disabled {
+  color: #808080 !important;
+}
+
+/* ── Sunken input fields, textareas & selects ─────────────────────────────*/
```

Stylelint (17.12.0)

[error] Expected whitespace before "*/" (comment-whitespace-inside)

(comment-whitespace-inside)